package.json

```diff
@@ -14,7 +14,7 @@
     "lint:fix": "yarn lint --fix",
     "create-modular-react-app": "ts-node packages/create-modular-react-app/src/cli.ts",
     "modular": "ts-node packages/modular-scripts/src/cli.ts",
-    "test": "yarn modular test --watchAll false --runInBand --env node",
+    "test": "yarn modular test --regex --watchAll false --runInBand --env node",
```

Comment:
How come you're adding `--regex` without an argument to this?

Reply:
Modular is "special": it needs to be tested with Modular but wants to be built in a custom way. `--regex` will be passed to Jest, which will test everything that matches its patterns in every package (empty string works like `*`), which, in this repo, contains non-modular tests too. This is a necessary trick until we are ready to build Modular with Modular.